docs/postmortems/README.md

```diff
@@ -0,0 +1,21 @@
+# Postmortems
+
+> A postmortem is a written record of an incident, its impact, the actions taken
+> to mitigate or resolve it, the root cause(s), and the follow-up actions to
+> prevent the incident from recurring.
+>
+> [Postmortem Culture: Learning from failure](https://landing.google.com/sre/sre-book/chapters/postmortem-culture/),
+> _Devin Carraway_
+
+For a full overview, checkout the Software Reliability Engineering book on this
+topic [here](https://landing.google.com/sre/sre-book/chapters/postmortem/).
```

Comment:
https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/#:~:text=Some%20links%20use%20the%20word,won't%20have%20a%20clue.

```suggestion
For a full overview, checkout the [Software Reliability Engineering book on this
topic](https://landing.google.com/sre/sre-book/chapters/postmortem/).
```

Reply:
🤦